modules/cluster-logging-deploy-console.adoc

```diff
@@ -18,22 +18,27 @@ You can install the {clo} by using the {product-title} web console.
 . In the {product-title} web console, click *Operators* -> *OperatorHub*.
 . Type `OpenShift Logging` in the *Filter by keyword* box.
 . Choose  *Red Hat OpenShift Logging* from the list of available Operators, and click *Install*.
-. Ensure that *A specific namespace on the cluster* is selected under *Installation mode*.
-. Ensure that *Operator recommended namespace* is *openshift-logging* under *Installed Namespace*.
-. Select *Enable operator recommended cluster monitoring on this namespace*.
+ifdef::openshift-rosa[]
+. In the *role ARN* text field, type "no_role_arn". Although the specific ARN for the cluster is not required, the *role ARN* text field must be populated to continue the installation process.
 +
-This option sets the `openshift.io/cluster-monitoring: "true"` label in the `Namespace` object. You must select this option to ensure that cluster monitoring scrapes the `openshift-logging` namespace.
```

Comment:
I don't this this stanza should be removed

Reply:
That is correct. The original ticket tied to this PR was to remove step 6 from the current docs b/c (according to the ticket reporter and evident in the UI) that the step was no longer applicable. 
![image](https://github.com/openshift/openshift-docs/assets/122639474/d4194575-6b58-4938-8014-165d48359bfa)